ocaml/xapi/quicktest_cbt.ml

```diff
@@ -49,27 +52,46 @@ let make_vdi_from ~session_id ~sR = (* SR has VDI.create as allowed *)
 
 (**** Test declarations ****)
 
-(* Test enable/disable CBT, test cbt_enabled:false for new VDI *)
-let enable_disable_cbt_test ~session_id ~vDI =
-  let enable_cbt_test = make_test "Testing VDI.enable/disable_CBT" 4 in
+(* Test data_destroy and snapshot update the necessary fields *)
```

Comment:
We also test here that `VDI.enable_cbt` and `VDI.disable_cbt` update the `cbt_enabled` field of the VDI accordingly when they return without errors.